pdns/recursordist/settings/table.py

```diff
@@ -2154,6 +2154,19 @@
 Throttle a server that has failed to respond :ref:`setting-server-down-max-fails` times for this many seconds.
  ''',
     },
+    {
+        'name' : 'server_down_use_probability',
```

Comment:
I wonder if it would make sense to use the word `throttled` here? Perhaps something like `bypass_server_throttling_probability`?